test/blackbox-tests/test-cases/stop-on-first-error.t/run.t

```diff
@@ -0,0 +1,19 @@
+Here we demonstrate the --stop-on-first-error behaviour.
```

Comment:
Can you change the test not to rely on OCaml? Simple rules and actions will do here.